.vscode/settings.json

```diff
@@ -42,7 +42,7 @@
   },
   "[typescriptreact]": {
     "editor.formatOnSave": true,
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "rvest.vs-code-prettier-eslint"
```

Comment:
why is this needed?

Reply:
Oh I didn't know that changed stuff for anyone but me. I'll change it back.

Reply:
@jamesbchao can you change this back again?  it looks like it is still there in latest commit